scripts/webpack/dev.servers.config.js

```diff
@@ -31,16 +31,15 @@ module.exports = {
   output: {
     filename: '[name].js',
     path: path.join(PROJECT_ROOT, 'dev'),
-    libraryTarget: 'commonjs',
-    publicPath: `http://localhost:${process.env.PORT}/static/`
```

Comment:
publicPath is used for things like jira images that the cache locally. this has been moved to the image file loader below for specificity & to avoid the future use case where `.node` native modules are loaded from publicPath